pkg/apis/fission.io/v1/validation.go

```diff
@@ -36,7 +36,7 @@ const (
 
 var (
 	validAzureQueueName = regexp.MustCompile("^[a-z0-9][a-z0-9\\-]*[a-z0-9]$")
-	validKafkaTopicName = regexp.MustCompile("^[a-z0-9][a-z0-9\\-._]*[a-z0-9]$")
+	validKafkaTopicName = regexp.MustCompile("^[a-zA-Z0-9][a-zA-Z0-9\\-._]*[a-zA-Z0-9]$")
```

Comment:
should be in `[a-zA-Z0-9][a-zA-Z0-9\\-\\._]*[a-zA-Z0-9]`

Reply:
Hey @life1347  ,
If we look at the existing regex, it seems to be matching `\` as well in the topic name. Please check this link https://regex101.com/r/Aa9MuI/1
I think we should use this `^[a-zA-Z0-9][a-zA-Z0-9\-\._]*[a-zA-Z0-9]$` instead

Reply:
@viveksinghggits Thanks for pointing that out. You're correct. 👍

Reply:
Thanks @viveksinghggits Great point!